pallets/capacity/README.md

```diff
@@ -0,0 +1,59 @@
+# Capacity Pallet
+
+Manages the staking and balances for Capacity, an alternative payment system on Frequency.
+
+## Summary
+
+Capacity is an alternative payment system for a subset of calls that renews each Epoch.
+Tokens staked create Capacity for a targeted Provider.
+[Learn more about Capacity](https://docs.frequency.xyz/Tokenomics/ProviderIncentives.html#capacity-model).
+
+### Staking & Unstaking
+Currently, the token to Capcity ratio is 50:1.
+For example, for a 5 token stake, a Provider would receive 0.1 Capacity.
+Staking and unstaking affect available Capacity immediately.
+
+### Capacity Epoch
+
+A Capacity Epoch is the number of blocks before Capacity balances are able to be renewed.
```

Comment:
Hmmm, "able to be renewed"? Shouldn't it just be "renewed"? Or maybe something like "is the number of blocks that must be formed between Capacity balance renewals"?

Reply:
It should not just be renewed because of the lazy renewal.  But perhaps, "...can be renewed (via the first Capacity transaction in a new Epoch)"

Reply:
Ah... forgot about lazy renewal.

Reply:
It would be the maximum not the minimum.

Committing a revised version of these two lines...

Reply:
Revision committed. Feel free to start a new thread if more revisions are suggested.